product-management/onboarding.md

```diff
@@ -38,3 +46,7 @@ layout: default
 - Work closely with the Product team to refine user stories, clarify requirements, and align on sprint goals.
 
 At the end of the first week of onboarding, you should be ready to lead the sprint meetings, outlining goals, priorities, and expectations for the upcoming sprint.
+
+#### Walkthrough Video
+ 
+[Project setup guide for PMs](https://www.loom.com/edit/e7e977685c7a4887942defd3b6a52ffc)
```

Comment:
A silo section is fine (though personally I would remove it), but it needs to be incorporated - the link to video - at the place - as part of onboarding where we think its going to be most relevant, maybe under #### 3. Product Development & Execution.

We also cannot make "Kizuna" data accessible to public like this, unless they are part of Better team, so they should only be able to access this video when they are part of Better org.

In a super ideal world, we should create a "working session" video to help someone see through what goes into setting up a project from scratch. The amount of learning and clarity one gets seeing things from scratch v/s seeing already existing is vastly different.